### PR TITLE
Don't add launch_prefix_url to .env

### PR DESCRIPTION
**Pull request by Himavanth**
## Description
This is already being handled in aio-cli-plugin-app

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Himavanth** [comment]
@meryllblanchet @sandeep-paliwal @shazron

File: generators/add-web-assets/exc-react/index.js [hunk 1]
```diff
@@ -73,11 +78,15 @@ class ExcReactGenerator extends Generator {
...
+    const launchPrefix = getCliEnv() === STAGE_ENV
+      ? 'https://experience-stage.adobe.com/?devMode=true#/custom-apps/?localDevUrl='
+      : 'https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl='
+
     utils.appendOrWrite(
       this,
       this.destinationPath(dotenvFilename),
       `## URL prefix used to run your application in the Adobe Experience Cloud Shell
-AIO_LAUNCH_URL_PREFIX="https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl="
+AIO_LAUNCH_URL_PREFIX="${launchPrefix}"
```
Comment (moritzraho) [inline generators/add-web-assets/exc-react/index.js:89]: Should we remove this one from the .env, in https://github.com/adobe/aio-cli-plugin-app/pull/396 you already set the default.
If someone inits an app in stage and then wants to switch to prod he needs to edit that line in the `.env`. If we don't include it then app plugin will be able to set the correct default

Comment (Himavanth) [reply generators/add-web-assets/exc-react/index.js:89]: I agree. These kind of overrides could just be documented instead of actually putting in .env.

Comment (moritzraho) [reply generators/add-web-assets/exc-react/index.js:89]: 👍 for doc so then feel free to remove the env var 🙂

**Review by moritzraho — APPROVED** [review]
lgtm, thanks for the change

**Pull request merged**